sdk/src/main/java/com/mapbox/maps/StyleObserver.kt

```diff
@@ -35,9 +40,11 @@ internal class StyleObserver(
    * NOTE : listener is invoked only once after successful style load.
    */
   fun setLoadStyleListener(
+    transitionOptions: TransitionOptions?,
```

Comment:
Maybe create another `setTransitionOptions` function for this? It's a little wired to set `TransitionOptions` in this function.

Reply:
why? setting transition options is tightly connected with loading a style and setting listeners imho